### PR TITLE
Sonarlint issue resolved

### PR DESCRIPTION
**Pull request by denduluriramasaimohanakhil**
dev:
  lint:0
  coverage:85%
OBAC:
  lint:0
  coverage:85%
Sonarlint issue removed

### CODE REVIEW

File: src/main/java/com/techsophy/tsf/runtime/form/controller/impl/FormDataControllerImpl.java [hunk 1]
```diff
@@ -46,9 +46,9 @@ private static Optional<String> getAclFilter(List<ACLDecision> aclDecisionList)
         return aclDecisionList
                 .stream()
                 .map(ACLDecision::getAdditionalDetails)
-                .map(a->Optional.ofNullable((Map<String,String>)a.get("runtime-form-app")))
-                .map(b->b.map(c->c.get("filters")))
-                .reduce((d,e)->d.flatMap(f->Optional.of(f+e.orElse(""))))
+                .map(additionaDetailsMap->Optional.ofNullable((Map<String,String>)additionaDetailsMap.get("runtime-form-app")))
+                .map(x->x.map(runtimeFormMap->Optional.ofNullable(runtimeFormMap.get("filters")).orElseThrow()))
```
Comment (VaibhavTechsophy) [inline src/main/java/com/techsophy/tsf/runtime/form/controller/impl/FormDataControllerImpl.java:50]: Define strings as constants

File: src/main/java/com/techsophy/tsf/runtime/form/service/impl/FormDataServiceImpl.java [hunk 2]
```diff
@@ -675,7 +675,7 @@ private PaginationResponsePayload getPaginationWithMongoAndRelations(String form
                 aggregationOperationsList.add(Aggregation.match(aclFilterCriteria));
             }
             prepareDocumentAggregateList(mappedArrayOfDocumentsName, relationKeysList, relationValuesList, aggregationOperationsList);
-            PaginationResponsePayload paginationResponsePayload1 = getPaginationWithMongoAndEmptySort(formId, sortBy, sortOrder, pageable, paginationResponsePayload, content, aggregationOperationsList);
+            PaginationResponsePayload paginationResponsePayload1 = getPaginationWithMongoAndEmptySort(formId, sortBy, sortOrder, pageable, paginationResponsePayload, new ArrayList<>(), aggregationOperationsList);
```
Comment (VaibhavTechsophy) [inline src/main/java/com/techsophy/tsf/runtime/form/service/impl/FormDataServiceImpl.java:678]: for what is this parameter used?

Comment (VaibhavTechsophy) [reply src/main/java/com/techsophy/tsf/runtime/form/service/impl/FormDataServiceImpl.java:678]: if this is not used than remove it

**Review by VaibhavTechsophy — CHANGES_REQUESTED** [review]

File: src/main/java/com/techsophy/tsf/runtime/form/controller/impl/FormDataControllerImpl.java (hunk 1 above)
Comment (denduluriramasaimohanakhil) [reply src/main/java/com/techsophy/tsf/runtime/form/controller/impl/FormDataControllerImpl.java:50]: yes @VaibhavTechsophy defining strings as constants only

**Pull request closed without merging**